prow/plugins/assign/assign.go

```diff
@@ -145,6 +150,22 @@ func handle(h *handler) error {
 	}
 	if len(toAdd) > 0 {
 		h.log.Printf("Adding %s to %s/%s#%d: %v", h.userType, org, repo, e.Number, toAdd)
+		invalidLogins, err := h.invalidLogins(org, repo, e.Number, e.IssueAuthor.Login, toAdd)
+		if err != nil {
+			return err
+		}
+		if len(invalidLogins) > 0 {
+			h.log.Printf("Invalid users %v requested to be added to %s/%s#%d", invalidLogins, org, repo, e.Number)
```

Comment:
nit: why do we use `Printf` in this handler? How does that even get routed? If this is application logging, `logrus.Debug`?

Reply:
Thank you for the review. As discussed on slack, I have updated the github client to handle 404 (which is how GitHub API responds to one or more non-existent logins in the assign issue request).